generators/app/templates/src/app/components/SearchBar/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+interface Props {
+  buttonClassName?: string;
+  className?: string;
+  formClassName?: string;
+  submitButtonText: string;
```

Comment:
May be this could be optional. As this is a search bar, some times we use image assets instead of a text for the submit button.

Reply:
Fixed both this and the next content by making this prop `submitButtonContent`, which accepts anything renderable. It might be an image, a text or whatever, but I think this should be mandatory now. I don't think an empty button makes sense.